config/default/kustomization.yaml

```diff
@@ -0,0 +1,74 @@
+# Adds namespace to all resources.
```

Comment:
Ignore the files generated for config for now, was mostly concentrated on the code for now.